src/components/Buttons.js

```diff
@@ -148,9 +148,21 @@ export function ShareButton(){
 }
 
 export function ArrowButton({ text }) {
+
+  const Text = styled(GradientText)`
+  font-family: Inter;
+  font-style: normal;
```

Comment:
components should not get nested like this, please move `Text` out of the `ArrowButton`